From 5b554b0a29ce9610e3c237c77a1f76db87454b72 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Tue, 16 Jun 2020 14:21:55 +0900
Subject: [PATCH] ASoC: remove snd_soc_component_read32()
Git-commit: 5b554b0a29ce9610e3c237c77a1f76db87454b72
Patch-mainline: v5.9-rc1
References: jsc#SLE-16518

No driver is using snd_soc_component_read32() anymore.
This patch removes it.

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Link: https://lore.kernel.org/r/877dw74mbv.wl-kuninori.morimoto.gx@renesas.com
Signed-off-by: Mark Brown <broonie@kernel.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 include/sound/soc-component.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/sound/soc-component.h b/include/sound/soc-component.h
index f64cffa12967..8917b15eccae 100644
--- a/include/sound/soc-component.h
+++ b/include/sound/soc-component.h
@@ -333,7 +333,6 @@ void snd_soc_component_set_aux(struct snd_soc_component *component,
 int snd_soc_component_init(struct snd_soc_component *component);
 
 /* component IO */
-#define snd_soc_component_read32 snd_soc_component_read
 unsigned int snd_soc_component_read(struct snd_soc_component *component,
 				      unsigned int reg);
 int snd_soc_component_write(struct snd_soc_component *component,
-- 
2.16.4

